src/pages/suggestions/suggestions.ts

```diff
@@ -192,22 +31,22 @@ export class SuggestionsPage {
 
   updateTopic() {
     console.log("topic " + this.topic);
```

Comment:
The huge `if` is still horrible, but I'm happier knowing that the data is out of there. I'll have a think about a nicer way of solving this.